apps/showcase/scripts/update-screenshots/podman-script.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Read first argument which is the list of ip addresses
+ipAddresses=$1
+
+echo "=> Entering podman container"
+
+# Find a working IP
+export PLAYWRIGHT_TARGET_URL=$(echo $ipAddresses | xargs -d, -r -I {} sh -c 'wget -T 2 -t 1 -q {} && echo {}' 2>null | sed 1q)
+
+# Exit if nothing works
+[ $PLAYWRIGHT_TARGET_URL ] || { echo 'The http-server cannot be reached with any of [$ipAddresses}]' >&2; exit 0; }
+
+# Copy local repo inside a temporary folder
+echo "Cloning local repo... (Don't forget to commit all the needed files for e2e)"
+git clone tests tests-clone
+
+# Move to the folder
+cd tests-clone
+
+# Delete the existing screenshots
+echo 'Delete previous screenshots'
+rm -rdf apps/showcase/e2e-playwright/sanity/screenshots
+
+# Configure the global cache to target a folder outside the container to have it available for the following runs
+yarn config set enableGlobalCache true
```

Comment:
Would be safer to do a `corepack enable` before (if not already done in the docker config)